views/includes/inputs/address/residential-address-input.html

```diff
@@ -100,6 +100,9 @@
       classes: "govuk-fieldset__legend--m"
     }
   },
+  hint: {
```

Comment:
residential address is on add pages too. Is this hint to for them?

Reply:
The hint is for "This is also known as a service address". Not currently on the add page

<img width="711" alt="Screenshot 2023-04-18 at 11 48 49" src="https://user-images.githubusercontent.com/118522865/232755053-9a441c18-8e32-41c3-8572-67b046801065.png">